bigquery/google/cloud/bigquery/table.py

```diff
@@ -473,8 +473,9 @@ def _build_resource(self):
             resource['schema'] = {
                 'fields': _build_schema_resource(self._schema)
             }
-        else:
-            raise ValueError("Set either 'view_query' or 'schema'.")
+        elif self.partitioning_type is None:
+            raise ValueError(
+                "Set either 'view_query' or 'schema' or 'partitioning_type'.")
```

Comment:
Oh, and I will change `elif self._schema:` to `if self._schema:` for the same reason: Not making assumptions the server side will never allow both a view query and a schema; leave the validation to the server side. OK?